### DISCUSSION TITLE
Kernel error AppleACPIPlatform

### DISCUSSION THREAD

**Discussion by AlinEusebiu98** [PIut02/ROG-Zephyrus-G14-GA401IV-Hackintosh#30] [Q&A]
Hello guys, I did try this with my 2021 GA401Q, I do get the same error every time I tryed, I did try a clean EFI made from 0, I did try this one and change the settings that you suggested in the readme part.
I did try with the recovery method (Monterey) and with the Sonoma.raw 14.x but never got to the installation part, I know it's something with the booting part, but can't figure it out, I did try everyting for the past 3 days...

**My specs:**
GA401QE.415 
Ryzen 7 5800HS with Vega 8
16GB RAM DDR4 = Samsung M471A1G44AB0
Nvidia RTX 3050Ti 4GB
SSD: SAMSUNG MZVLQ512HBLU
Wi-Fi: MediaTek MT7921

![WhatsApp Image 2024-02-10 at 11 36 15_8a83d0ea](https://github.com/PIut02/ROG-Zephyrus-G14-GA401-Hackintosh/assets/74622198/65a806dc-048b-489c-bb17-e45d494a8780)
![WhatsApp Image 2024-02-10 at 11 41 38_b6a074fb](https://github.com/PIut02/ROG-Zephyrus-G14-GA401-Hackintosh/assets/74622198/07215110-ee6d-4826-a173-6b3f9066c699)

**Comment by PIut02** [comment]
Try clearing NVRAM

**Reply by AlinEusebiu98** [reply]
I did, every single time a try others EFI's.

**Reply by PIut02** [reply]
Maybe this model has some special settings, try to modify it according to the method of this website: https://dortania.github.io/OpenCore-Install-Guide/extras/kaslr-fix.html#using-devirtualisemmio

**Reply by AlinEusebiu98** [reply]
I did try, I also did try the NormalizeHeaders. Same error.

**Reply by PIut02** [reply]
I think this problem may be caused by ssd, try replacing it

**Reply by AlinEusebiu98** [reply]
At some point I get stuck at AE_ALREADY_EXIST i did get the SSDT-CPUR.aml and now I get stuck here, it s rebooting after this, but can t se any error.

![image](https://github.com/PIut02/ROG-Zephyrus-G14-GA401-Hackintosh/assets/74622198/e30c5cf1-b4ac-45b8-9407-8dc7b3ab0602)
